test/integration/replicaset/test_replicaset_roles_add.py

```diff
@@ -0,0 +1,296 @@
+import io
+import os
+import shutil
+
+import pytest
+from cartridge_helper import cartridge_name
```

Comment:
Up to you.
```suggestion
import pytest

from cartridge_helper import cartridge_name
```

Reply:
It will be a lint error in this case:
```
test/integration/replicaset/test_replicaset_roles_add.py:6:1: I004 isort found an unexpected blank line in imports
```

Reply:
Ok, thank you.